tests/fixtures.py

```diff
@@ -296,6 +298,26 @@ def reset(self):
     return _CapsuleSideChannel()
 
 
+@pytest.fixture(scope="module")
+def capsule_side_channel_blockchain(enacted_blockchain_policy):
```

Comment:
Interesting - What Incompatibility did you encounter with the Federated Side Channel?

Reply:
I think, they have different labels, so it pretty much appears to be the wrong capsule (and yielding a decryption error). It's not incompatibility per se, I think fixtures can be worked on. I just didn't focus on that in this PR - wanted them minimally working.